### PR TITLE
Feat/action modal

### PR DESCRIPTION
**Pull request by kmelkon**

### CODE REVIEW

**Review by ratkosolar — APPROVED** [review]

File: src/Atoms/Illustration/IllustrationBase.types.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export type IllustrationProps = {
   title?: string;
   color?: ColorFn | string;
   secondaryColor?: ColorFn | string;
+  primaryColor?: ColorFn | string;
```
Comment (jesperwiner) [inline src/Atoms/Illustration/IllustrationBase.types.ts:17]: there is no need for this, "color" is the primary color already

File: src/Molecules/ActionModal/ActionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+              <Flexbox
+                container
+                direction={isSmallerScreen ? 'column-reverse' : 'row'}
+                justifyContent="space-between"
+                gap={3}
+              >
+                <Button variant="neutral" onClick={onSkipClick}>
+                  <Typography color={(t) => t.color.actionModalLink} type="secondary" weight="bold">
+                    Skip the tour
```
Comment (jesperwiner) [inline src/Molecules/ActionModal/ActionModal.tsx:94]: hardcoded texts?

File: src/Atoms/Illustration/IllustrationBase.types.ts (hunk 1 above)
Comment (kmelkon) [reply src/Atoms/Illustration/IllustrationBase.types.ts:17]: i didnt add the primarycolor, only the secondary one so im hesutant to remove it

File: src/Molecules/ActionModal/ActionModal.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  children,
+  onSkipClick,
+  onCTAClick,
+}: {
+  autoFocus: boolean;
+  illustration: any;
+  title: string;
+  children: any;
+  onSkipClick: () => void;
+  onCTAClick: () => void;
```
Comment (jesperwiner) [inline src/Molecules/ActionModal/ActionModal.tsx:69]: maybe. "onClick". and "onClose" instead. or "onSecondaryClick"

**Review by ratkosolar — DISMISSED** [review]

**Review by jesperwiner — APPROVED** [review]

**Review by alvelid — APPROVED** [review]

**Pull request merged**